code/DefaultMembers.php

```diff
@@ -73,4 +81,39 @@ public static function create_default_members($silent=FALSE) {
         }
     }
 
+    /**
+     * Remove default members
+     *
+     * @param bool $silent Pass true to hide `/dev/build` messages
+     */
+    public static function remove_default_members($silent = false)
+    {
+        $remove = Config::inst()->get('DefaultMembers', 'remove');
+
+        if ($remove && is_array($remove)) {
+            foreach ($remove as $data) {
+
+                if (!Email::is_valid_address($data['Email'])) {
+                    continue;
+                }
+
+                $member = Member::get()
+                    ->filter('Email', $data['Email'])
+                    ->first();
+
+                if ($member && $member->exists()) {
+                    $member->delete();
+
+                    /**
+                     * Display the change
+                     */
+                    if (!$silent) {
+                        DB::alteration_message("Removed $member->Name ($member->Email).", "deleted");
```

Comment:
I don't think you can reference `$member->Name` after you've deleted the member.

Reply:
At this point, the DB record is removed but the object should still exist until it is unset.

Reply:
I stand corrected! :)